src/qililab/slurm.py

```diff
@@ -0,0 +1,111 @@
+import ast
+import os
+from types import ModuleType
+
+from IPython.core.magic import needs_local_scope, register_cell_magic
+from IPython.core.magic_arguments import argument, magic_arguments, parse_argstring
+from submitit import AutoExecutor
+
+from qililab.config import logger
+
+num_files_to_keep = 20  # needs to be a multiple of 4 and 5: 20,40,60,80..
```

Comment:
Here you use 20, but in the tests you say 60... Can we increase it to 60?

Reply:
done!